lld/test/ELF/keep-text-section-prefix.s

```diff
@@ -0,0 +1,89 @@
+# REQUIRES: x86
+
+# RUN: rm -rf %t && split-file %s %t && cd %t
+
+# RUN: llvm-mc -filetype=obj -triple=x86_64-pc-linux input.s -o a.o
+
+# RUN: ld.lld -z keep-data-section-prefix -T linker-script.lds a.o -o a.out
+# RUN: llvm-readelf -l a.out | FileCheck --check-prefixes=SEG,LS %s
+# RUN: llvm-readelf -S a.out | FileCheck %s --check-prefix=CHECK-LS
+
+# RUN: ld.lld -z keep-data-section-prefix a.o -o b.out
```

Comment:
Our convention is like `a.o => a`, `b.o => b`. The output file names `out` `out1` `out2` are more common.